cwms-data-api/src/test/java/cwms/cda/helpers/DTOMatch.java

```diff
@@ -69,6 +69,13 @@ public static void assertMatch(StreamLocation streamLocation, StreamLocation des
         );
     }
 
+    public static void assertMatch(StreamLocationNode streamLocationNode, StreamLocationNode streamLocationNode1) {
+        assertAll(
+            () -> assertMatch(streamLocationNode.getId(), streamLocationNode1.getId()),
```

Comment:
add message

Reply:
added messages where applicable. The messaging for CwmsIds are handled internally inside the assertMatch method for CwmsIds.